website/static/js/search.js

```diff
@@ -145,14 +205,43 @@ var ViewModel = function(params) {
         };
     });
 
+    self.filters = function(){
+        var licenses = self.selectedLicenses();
+        if (licenses.length) {
+            var filters = {
+                terms: {
+                    'license.name': licenses
+                }
+            };     
+            if (licenses.filter(function(l) {
+                return l.name === 'None Selected';
```

Comment:
If this returns `true`, will `fullQuery` behave correctly? What happens when `query.filtered.filter = true`?